packages/astro/src/core/config/schema.ts

```diff
@@ -186,6 +187,7 @@ export const AstroConfigSchema = z.object({
 				.enum(['always', 'auto', 'never'])
 				.optional()
 				.default(ASTRO_CONFIG_DEFAULTS.build.inlineStylesheets),
+			concurrency: z.number().min(1).optional().default(ASTRO_CONFIG_DEFAULTS.build.concurrency),
```

Comment:
@bluwy do you think we should set a `max`?

Reply:
Setting a max would be opinionated I think so probably best to leave it out for now I think